src/main/java/lotto/machine/LottoMachine.java

```diff
@@ -0,0 +1,44 @@
+package lotto.machine;
+
+import lotto.system.InputView;
+import lotto.system.OutputView;
+import lotto.ticket.LottoTicket;
+import lotto.ticket.LottoTickets;
+
+public class LottoMachine {
+    private static int LOTTO_AMOUNT = 1000;
+
+    public static void gameStart(){
+
+        OutputView.printInputMoney();
+        String money = InputView.inputMoney();
+
+        LottoTickets buyLottoTickets = buyLotto(Integer.parseInt(money));
+
+        OutputView.printInputWinningNumbers();
+        String winningNumbers = InputView.inputWinningNumbers();
+        LottoTicket winningLottoTicket  = new LottoTicket(winningNumbers);
+
+        Result result = buyLottoTickets.match(winningLottoTicket);
+        result.setMoney(Integer.parseInt(money));
+        printResult(result);
```

Comment:
```suggestion
        OutputView.printResult(result);
```
단순 같은 클래스 메서드 위임이라, 이렇게 사용해도 괜찮을 것 같아요 :)
제 생각이니 변경은 하지 않아도 됩니다 ㅎㅎ